alacritty_terminal/src/ansi.rs

```diff
@@ -1053,7 +1061,7 @@ where
             // Reset color index.
             b"104" => {
                 // Reset all color indexes when no parameters are given.
-                if params.len() == 1 {
+                if params.len() > 1 && params[1].is_empty() {
```

Comment:
Looks like you've just changed it from one incorrect implementation to another here. `OSC 104 ST` and `OSC 104 ; ST` should **both** work. At least I'm pretty sure that's what XTerm does.